### PR TITLE
docs(rxMetadata): Page objects to jsdoc3

### PR DESCRIPTION
**Pull request by Droogans**
### LGTMs
- [x] Dev LGTM
- [x] QE LGTM

### CODE REVIEW

File: src/components/rxMetadata/scripts/rxMetadata.exercise.js [hunk 1]
```diff
@@ -3,20 +3,19 @@ var rxMetadata = require('./rxMetadata.page').rxMetadata;
...
+ * @param {Object} [options.transformFns] - Transformation functions to be passed to rxMetadata.
+ * @param {Object} [options.terms] The expected label text of each metadata entry.
  * @example
- * ```js
  * describe('metadata', encore.exercise.rxMetadata({
  *     transformFns: {
  *         'Signup Date': function (elem) {
  *             return elem.getText().then(function (text) {
- *                 return new Date(text);
+ *                 return new Date(text).valueOf();
```
Comment (CITguy) [inline src/components/rxMetadata/scripts/rxMetadata.exercise.js]: What's the significance of using `valueOf()` in this example?

Comment (Droogans) [inline src/components/rxMetadata/scripts/rxMetadata.exercise.js]: It's a testing quirk that I tried to keep true in form. Dates are not simple to compare for equality.

```js
// this fails
expect(promiseWithSomeDate).to.eventually.eql(new Date('2016-01-01'));
expect(promiseWithSomeDate).to.eventually.equal(new Date('2016-01-01'));

// all these fail too
expect(new Date('2016-01-01')).to.eql(new Date('2016-01-01'));
expect(new Date('2016-01-01')).to.equal(new Date('2016-01-01'));

// this works
expect(new Date('2016-01-01').getYear()).to.equal(new Date('2016-01-01').getYear());
expect(new Date('2016-01-01').getMonth()).to.equal(new Date('2016-01-01').getMonth());
// etc.

// or, more succinctly
expect(new Date('2016-01-01').valueOf()).to.equal(new Date('2016-01-01').valueOf());
```

[chai datetime](http://chaijs.com/plugins/chai-datetime) helps, but it's on you to install it. I try not to assume folks have chai plugins when writing documentation examples.

Comment (CITguy) [inline src/components/rxMetadata/scripts/rxMetadata.exercise.js]: Ok, thanks for the explanation.

**Comment by jj4th** [comment]
QE-LGTM

**Comment by CITguy** [comment]
LGTM

**Pull request merged**